build_tools/bazel/iree.bazelrc

```diff
@@ -298,10 +298,11 @@ build:windows --define=override_eigen_strong_inline=true
 # Another TensorFlow flag from their config script.
 build:windows --define with_default_optimizations=true
 
-# TensorFlow builds depend on these flags so we will as well
+# TensorFlow builds depend on this flag, but it doesn't appear to work with
+# gmock in some of our unit tests, so only enable it for TensorFlow files.
 # MSVC (Windows): Standards-conformant preprocessor mode
 # See https://docs.microsoft.com/en-us/cpp/preprocessor/preprocessor-experimental-overview
-build:windows --copt=/experimental:preprocessor
-build:windows --host_copt=/experimental:preprocessor
+build:windows --per_file_copt=tensorflow@/experimental:preprocessor
+build:windows --per_file_copt=tensorflow@/experimental:preprocessor
```

Comment:
This is duplicated now? Is `per_file_host_copt` a thing? Or does plain `host_copt` work? or should we delete one of these?

Reply:
Oh, good catch lol. Let me check. The `host_copt` might not actually be needed then :P

Reply:
Let's just use `per_file_copt` for now and look deeper if needed. I don't see `per_file_host_copt` (or something like it) on https://docs.bazel.build/versions/master/user-manual.html. Might not be an issue if we aren't cross compiling on Windows with Bazel.